css/src/alerts.scss

```diff
@@ -69,20 +69,27 @@ h3 .dashicons-warning {
 .restore .dashicons,
 .dismiss .dashicons {
 	font-size: 24px;
+	width: 24px;
+	height: 24px;
 }
 
 .yoast-bottom-spacing {
 	margin-bottom: 20px;
 }
 
-.restore,
-.dismiss {
-	width: 32px;
+.yoast-alerts .button.restore,
+.yoast-alerts .button.dismiss {
+	flex: 0 0 40px;
+	width: 40px;
 	height: 40px;
-	margin-left: 8px;
-	border: none;
-	background-color: transparent;
-	cursor: pointer;
+	line-height: inherit;
+}
+
+.yoast-alerts .button.restore:hover,
```

Comment:
The reason why I've used nested selectors is to have a higher specificity than `.wp-core-ui .button` otherwise some properties (see for example the height) would be overridden. This is one of the things a single CSSBEM selector can't solve unless we want to start using `!important` everywhere, which is even worse. CSSBEM makes sense when it's used consistently across a whole application, but we're in a mixed environment here. I agree that overqualified selectors should be avoided as well, but unfortunately WP core uses `.wp-core-ui .button` and we need to override that.
Happy to follow your final decision, but between the two options my personal opinion is that `!important` is evil and I'd recommend to avoid it at all costs :)

Maybe they could be reduced to e.g. `.yoast-alerts .dismiss` if we're 100% sure the alerts css will be always loaded after buttons.css